components/Dialog/EditMetadata.vue

```diff
@@ -109,18 +135,34 @@ const isDate = (prop: any) => {
   return prop?.type == 'string' && prop?.format == 'date';
 };
 
+const isDateTime = (prop: any) => {
+  return prop?.type == 'string' && prop?.format == 'date-time';
+};
+
+const isSimpleString = (prop: any) => {
+  return prop?.type == 'string' && !prop?.format && !prop?.enum;
+};
+
 const isCheckbox = (prop: any) => {
   return prop?.type == 'boolean';
 };
 
 const updateMetadata = async () => {
+  const formattedMetadata: typeof metadata.value = {};
+  Object.entries(metadata.value).forEach(([k, v]) => {
+    formattedMetadata[k] = v;
+    if (isDate(properties[k]) || isDateTime(properties[k])) {
+      formattedMetadata[k] = new Date(v as string).toISOString();
```

Comment:
ISO 8601 string is the valid format for `date-time` whereas simply `YYYY-MM-DD` is valid for `date` 

`date` was handled correctly, only `date-time` has to be changed